source/security/container-security.rst

```diff
@@ -0,0 +1,98 @@
+Container Security
+------------------
+
+Container in |SPN| is a method for sub region support. Its an encapsulation of multiple components as depicted in the image below.
+
+.. image:: /images/sbl_container.PNG
+
+**Container**
+
+Container region header encapsulates details about its components and public key hashes or component hashes. Container header has to be signed using RSA private key. Respective public key hash would be store in |SPN| Hash Store and its verified while container gets registered.
+
+**Components**
+
+Component region can be added as compressed form or raw images. These region supports both hash verification and siging with RSA private keys. Public key hash or component hash would be stored in container main header. These hashes would be verified when components gets loaded.
+
+**Mono Signing**
+
+Mono signing is a method in which all components region would be verified using RSA or with hash verification.

```

Comment:
Mono signing is a method in which all components region would be signed in a single shot and verified using RSA or with hash verification.